SYCL/GroupAlgorithm/SYCL2020/sort.cpp

```diff
@@ -349,6 +349,9 @@ int main(int argc, char *argv[]) {
     return 0;
   }
 
+  bool doubleIsSupported = q.get_device().has(sycl::aspect::fp64);
+  bool halfIsSupported = q.get_device().has(sycl::aspect::fp16);
+
```

Comment:
I would inline those, but it's subjective.

Reply:
Inlined in a5e1dc556260b085785915c8ef8cc4f432252dce